### ISSUE TITLE
A11y_AzureDataStudio_ServiceSettings_AI4D:Name property does not exist for the checkbox.

### ISSUE THREAD

**Issue by vinodbk197** [microsoft/azuredatastudio#7605]
"[Check out Accessibility Insights!](https://accessibilityinsights.io) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

**GitHubTags:**#A11y_AzureDataStudio;#A11yMAS; #A11yTCS;#SQL Azure Data Studio;#WCAG1.3.1;#WCAG4.1.2;#WCAG4.2.1;#DesktopApp;#NVDA;#JAWS;#AI4D;#Win32;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.12.0-insider (user setup)
Commit: 32df727
Date: 2019-10-08T19:18:43.743Z
VS Code: 1.38.0
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362

**Additional Details:**
Tool Name : AI4D

**Repro Steps:**

1.	Install /Launch the application Azure data studio.
2.	Navigate to connections and click on more action button.
3.	Select “Deploy SQL Server big data cluster” from more action drop down.
4.	Click on “Deploy SQL Server big data cluster”.
5.	Check the check box I accept in the “select the deployment option” screen and click on select button.
6.	Select any block and click on next button and again click on next button.
7.	Enter password and confirm password and click on next button.
8.	Open AI4D and check the name property for the checkbox.

**Actual:**
Name property does not exist for the "Enable  availability groups" Checkbox.

**Expected:**
Name property should provide for the Checkbox as "Enable  availability groups" and screen reader should announces the same.

**User Impact:**
Screen reader user does not get name for the regarding the control.

**Attachment for Reference**:
[7605_A11y_AzureDataStudio_ServiceSettings_AI4D.pptx](https://github.com/microsoft/azuredatastudio/files/3712446/7605_A11y_AzureDataStudio_ServiceSettings_AI4D.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by alanrenmsft** [comment]
now that we no longer have that particular checkbox due to design change, but we still one checkbox and we have fixed the issue

<img width="1159" alt="Screen Shot 2019-11-24 at 12 52 28 PM" src="https://user-images.githubusercontent.com/13777222/69501399-61c10600-0eb9-11ea-8097-7459ca6adea1.png">

![Screen Shot 2019-11-24 at 12 52 34 PM](https://user-images.githubusercontent.com/13777222/69501400-61c10600-0eb9-11ea-8146-d6f120788cfa.png)

**Comment by vinodbk197** [comment]
Verified the issue on:
Azure Data Studio
Version: 1.15.0-insider (user setup)

The issue is resolved,Hence closed this issue.
Please refer the below attachment:
![image](https://user-images.githubusercontent.com/55909580/72730904-7a4b5500-3bb8-11ea-8664-372f6b1ae50c.png)

**Comment by fsteffi** [comment]
#Azure Data Studio - New Features_win32;

**Comment by Yash14j** [comment]
#A11ysev2;#Closed;
